src/Microsoft.IdentityModel.Tokens/SecurityTokenDescriptor.cs

```diff
@@ -94,6 +94,16 @@ public class SecurityTokenDescriptor
         /// </summary>
         public IDictionary<string, object> AdditionalInnerHeaderClaims { get; set; }
 
+        /// <summary>
+        /// Gets or sets the <see cref="HashSet{T}"/> which contains the default header claims that should be excluded from the JWT header.
+        /// </summary>
+        /// <remarks>
+        /// The default header claims are 'alg', 'kid', 'x5t', 'enc', and 'zip'.
```

Comment:
and by default, they are not excluded.?

Reply:
By default this set is empty so the default header claims that apply are included.